usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -62,14 +63,26 @@ def test_sles_hardened(host, get_release_value, is_sles_sap, is_sle_micro):
         )
     )
 
+    scap_report = os.environ.get("SCAP_REPORT", "")
+    if scap_report:
+        if os.path.exists(scap_report) or not scap_report.endswith(".html"):
```

Comment:
Sorry, not clear the aim of check `exists`: why `if file exists` it is `not good` and skipped ? 
We could append more text to the log, in case already used elsewhere?

Reply:
Moreover, if `not exist`, that new file is created

Reply:
> Sorry, not clear the aim of check `exists`: why `if file exists` it is `not good` and skipped ? 

We want to avoid any overwriting.

> Couldn't we append more text to the log,i.e. in case already used elsewhere?

What is the likely case that it happens?  Does it justify making the code more complicated because of overengineering?

Reply:
> If a previous report can never exist, `already exist` will never happen, then ok.

As of now, we're the only ones interested in this report that we consume in openQA.